hazelcast/src/main/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/IOBalancer.java

```diff
@@ -117,13 +117,13 @@ public void connectionRemoved(Connection connection) {
         }
 
         TcpIpConnection tcpIpConnection = (TcpIpConnection) connection;
-        NonBlockingReadHandler readHandler = (NonBlockingReadHandler) tcpIpConnection.getReadHandler();
+        NonBlockingSocketReader readHandler = (NonBlockingSocketReader) tcpIpConnection.getSocketReader();
```

Comment:
Another `NonBlockingSocketReader readHandler` and `NonBlockingSocketWriter writeHandler`.